### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
[DEMO LINK](https://andriy-shymkiv.github.io/react_movies-list/)

### CODE REVIEW

**Review by Anna-Shapovalova — APPROVED** [review]
Looks good to me) Well done) 👍  Maybe you should add also .scss files for MovieCard and MovieList...

File: src/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -2,6 +2,62 @@ import React from 'react';
...
+        <div className="content">
+          <p data-cy="MovieDescription">
+            {description}
+          </p>
+
+          <a
+            href={imdbUrl}
+            data-cy="MovieLink"
+          >
+            {imdbId}
```
[the remark this answers is not in the document]
Comment (Andriy-Shymkiv) [reply src/components/MovieCard/MovieCard.tsx:57]: got it

File: src/components/MovieCard/MovieCard.tsx [hunk 2]
```diff
@@ -2,6 +2,62 @@ import React from 'react';
...
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
```
Comment (illiarerun) [inline src/components/MovieCard/MovieCard.tsx:17]: 'imdbId' is assigned a value but never used, do you really need it here?

File: src/components/MovieCard/MovieCard.tsx [hunk 3]
```diff
@@ -2,6 +2,62 @@ import React from 'react';
...
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            // eslint-disable-next-line max-len
```
Comment (illiarerun) [inline src/components/MovieCard/MovieCard.tsx:26]: why?
```suggestion
```

**Review by illiarerun — CHANGES_REQUESTED** [review]
deploy new version of your solution
image

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**